packages/graphql-cli-graphback-plugin/README.md

```diff
@@ -0,0 +1,35 @@
+## Graphback
+
+<p align="center">
+  <img width="400" src="https://github.com/aerogear/graphback/raw/master/website/static/img/graphback.png">
+  <br/>
+  Auto generate database structure, <br/>
+  GraphQL Resolvers and Queries from GraphQL types 🚀
+</p>
+
+**Documentation**: [https://graphback.dev](https://graphback.dev)
+**Repository**: [https://github.com/aerogear/graphback/](https://github.com/aerogear/graphback/)
+
+## Motivation 
```

Comment:
We are actually missing installation steps tha we had initially. Let's add them.

Reply:
I've updated it, let me know if that's any better